app/models/message/letter_with_invoice.rb

```diff
@@ -32,6 +35,7 @@ class Message::LetterWithInvoice < Message::Letter
 
   belongs_to :invoice_list
   serialize :invoice_attributes, Hash
+  validate :assert_valid_invoice_items
```

Comment:
Das Sentry-Issue das diesen Fehler ausweist, enthält die Fehlermeldung `RuntimeError: invoice invalid`. Dasselbe Problem kann also auch auftreten, wenn die Invoice aus einem anderen Grund invalid ist (auch aus zukünftigen Gründen, wenn die Invoices noch komplexer werden). Wie wäre es also, wenn wir hier statt Wissen über Invoice Items einzubauen folgendes machen würden:
```
validates_associated :invoice
```

Reply:
Notizen aus Besprechung: `invoice` ist keine Association, sondern wird on the fly aus serialisierten Attributen konstruiert (für mich ist nicht klar warum, aber ist eine andere Geschichte).
Die Lösung wäre also eher:
```
validate :assert_valid_invoice

def assert_valid_invoice
  errors.add(something something) unless invoice.valid?
end
```

Reply:
das funktioniert leider so nicht :cry: 

beim Validieren von Invoice kommt:  "Empfänger Addresse oder E-Mail muss ausgefüllt werden"

ich lasse es mal so wie es ist.